### PR TITLE
new helpers

### PR DESCRIPTION
**Pull request by obernardovieira**
This new methods are needed on mobile app.

### CODE REVIEW

**Comment by psoaresbj** [comment]
LGTM.

Just wondering @obernardovieira don't you want to add these helpers to README?

**Comment by obernardovieira** [comment]
> Just wondering @obernardovieira don't you want to add these helpers to README?

ahh damn, forgot.

What about using something like https://esdoc.org/ ?

**Comment by psoaresbj** [comment]
> > Just wondering @obernardovieira don't you want to add these helpers to README?
> 
> ahh damn, forgot. Thank you.
> 
> What about using something like https://esdoc.org/ ?

Or maybe [jsdoc](https://jsdoc.app/)

See trend list [here](https://openbase.com/categories/js/best-javascript-documentation-generator-libraries)

**Review by psoaresbj — APPROVED** [review]

**Comment by psoaresbj** [comment]
@obernardovieira WDYT about adding small examples of each method to the example app?

File: README.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ yarn add @impact-market/utils
 ### Available Methods:
 
 - `getWalletsBalance`
+- `estimateCommunityRemainFunds`
+- `frequencyToNumer`
+- `frequencyToText`
+- `toNumber`
+- `toToken`
```
Comment (psoaresbj) [inline README.md:21]: Wouldn't be better to separate the methods from the helpers

```markdown

### Available Helpers

- `estimateCommunityRemainFunds`
- `frequencyToNumer`
- `frequencyToText`
- `toNumber`
- `toToken`
```

**Comment by obernardovieira** [comment]
I'm merging this as it was approved before

**Pull request merged**